data/dsql/sql.go

```diff
@@ -19,14 +24,14 @@ import (
 // Additionally, if the SQLStringConverter's Replacer is not nil, the replacement will be performed.
 // A map of Field/Column index to the corresponding SQLStringConverter is returned so what conversions were
 // done can be inspected.
-func NewFromSQLRows(rows *sql.Rows, converters ...SQLStringConverter) (*Frame, map[int]SQLStringConverter, error) {
+func NewFromSQLRows(rows *sql.Rows, converters ...SQLStringConverter) (*data.Frame, map[int]SQLStringConverter, error) {
```

Comment:
Nit. Is SQL here needed now? This is currently the only exported function. Maybe rename to FromRows or NewFrameFromRows or FrameFromRows to denote that's a frame is created?